application/classes/Ushahidi/Repository/Post.php

```diff
@@ -867,6 +875,11 @@ public function create(Entity $entity)
 			$this->updatePostStages($id, $entity->form_id, $entity->completed_stages);
 		}
 
+		// TODO: Revist post-Kohana
+		// This might be better placed in the usecase but
+		// given Kohana's future I've put it here
+		$this->emit($this->event, $id, 'create');
```

Comment:
I kinda think we could embed these directly in the CRUDS usecases so they're universal.
But yea, we're likely to need to refactor anyway